### PR TITLE
Add cloudtrail event source page

### PR DESCRIPTION
**Pull request by mstemm**
It mostly points to other repos/readmes/etc for the details on how to
use Cloudtrail with falco, but we didn't really have a docs page about
Cloudtrail other than inside the plugins section, which is too much
detail for someone just trying to get cloudtrail to work.

Signed-off-by: Mark Stemm <mark.stemm@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/event-sources/_index.md [hunk 1]
```diff
@@ -3,8 +3,16 @@ title: Event Sources
 weight: 5
 ---
 
-Falco can consume events from different sources, and apply rules to these events to detect abnormal behavior. Currently Falco supports the following event sources:
+Falco can consume events from a variety of different sources and apply rules to these events to detect abnormal behavior.
+
+Currently Falco supports the following built-in event sources:
```
Comment (radhikapc) [inline content/en/docs/event-sources/_index.md:8]: Currently, Falco supports the following built-in event sources:

File: content/en/docs/event-sources/cloudtrail.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* A SQS queue that passes along SNS notifications about new log files
+* A local filesystem path
+
+Details on the open params syntax can be found on the [open params](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#plugin-open-params) section of the Cloudtrail README page.
+
+# Terraform Module For Cloudtrail Prerequsites
+
+In order to use the Cloutrail plugin, you must [enable](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-cloudtrail-logging-for-s3.html) Cloudtrail logging for the account(s) you want to monitor. This must be done before using the plugin.
+
+In addition, of the 3 options above, using a SQS queue provides the easiest-to-consume source of logs, as the plugin can be notified as new log files are written and automatically consume them. However, this also requires creating multiple AWS cloud resources such as SQS queues, SNS topics, etc. outside of Falco, which involve multiple manual steps.
```
Comment (radhikapc) [inline content/en/docs/event-sources/cloudtrail.md:32]: In addition, of the three options above, using an SQS queue provides the easiest-to-consume source of logs. With the SQS queue, the plugin can detect when the new log files are written and can automatically consume them. However, this also requires creating multiple AWS cloud resources, such as SQS queues, SNS topics, etc., outside of Falco, which involve multiple manual steps.

File: content/en/docs/event-sources/okta.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Falco also distributes out-of-the-box [rules](https://github.com/falcosecurity/falco/blob/master/rules/okta_rules.yaml) that can be used to identify interesting/suspicious/notable events in Okta logs, including:
+
+* Creating a new OKTA user account
+* Detecting a locked-out user
+* Assigning admin permissions to an okta user
+
+# Configuration
+
+The [README](https://github.com/falcosecurity/plugins/blob/master/plugins/okta/README.md#settings) page has full details on how to configure the plugin. This simply involves providing the organization/api token as a part of init params. These can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.
```
Comment (radhikapc) [inline content/en/docs/event-sources/okta.md:16]: The [README](https://github.com/falcosecurity/plugins/blob/master/plugins/okta/README.md#settings)  helps configure the plugin. This simply involves providing the `organization/api` token as part of init params. These can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.

File: content/en/docs/event-sources/okta.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+The plugin does not use any open params configuration.
+
+# Sample Output
+
+For a dummy rule like the following:
```
Comment (radhikapc) [inline content/en/docs/event-sources/okta.md:22]: For example:

File: content/en/docs/event-sources/okta.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- rule: Dummy
+  desc: Dummy
+  condition: okta.app!=""
+  output: "evt=%okta.evt.type user=%okta.actor.name ip=%okta.client.ip app=%okta.app"
+  priority: DEBUG
+  source: okta
+  tags: [okta]
+```
+
+For each log entry you would see the following falco alert:
```
Comment (radhikapc) [inline content/en/docs/event-sources/okta.md:34]: For each log entry for the above configuration,  you can see the following Falco alert:

**Review by radhikapc — CHANGES_REQUESTED** [review]
a few suggestions

File: content/en/docs/event-sources/_index.md (hunk 1 above)
Comment (mstemm) [reply content/en/docs/event-sources/_index.md:8]: Sure, fixed.

File: content/en/docs/event-sources/cloudtrail.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+Falco also distributes out-of-the-box [rules](https://github.com/falcosecurity/falco/blob/master/rules/aws_cloudtrail_rules.yaml) that can be used to identify interesting/suspicious/notable events in Cloudtrail logs, including:
+
+* Console logins that do not use multi-factor authentication
+* Disabling multi-factor authentication for users
+* Disabling encryption for S3 buckets.
+
+# Configuration
+
+The [README](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#configuration) page has full details on how to configure the plugin. The plugin initialization and open params strings/objects can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.
```
[the remark this answers is not in the document]
Comment (mstemm) [reply content/en/docs/event-sources/cloudtrail.md:16]: I fixed the initial part of the paragraph, but the term "open params" is a specific falco configuration item--see https://falco.org/docs/configuration/, specifically the `plugins` config item.

File: content/en/docs/event-sources/cloudtrail.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+# Ways to read Cloudtrail Logs
+
+The plugin can be configured to read log files from:
+
+* A S3 bucket
+* A SQS queue that passes along SNS notifications about new log files
+* A local filesystem path
+
+Details on the open params syntax can be found on the [open params](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail#plugin-open-params) section of the Cloudtrail README page.
```
[the remark this answers is not in the document]
Comment (mstemm) [reply content/en/docs/event-sources/cloudtrail.md:26]: Sure, fixed.

File: content/en/docs/event-sources/cloudtrail.md (hunk 2 above)
Comment (mstemm) [reply content/en/docs/event-sources/cloudtrail.md:32]: I mostly agree with this, but want to keep the "SNS topics" in there as it emphasizes that multiple resources are required. I'll actually add some additional resources to further emphasize this.

File: content/en/docs/event-sources/okta.md (hunk 4 above)
Comment (mstemm) [reply content/en/docs/event-sources/okta.md:22]: I want to make it clear that the output depends on the rule used to filter the events, so I ended up with "For example, when using a dummy rule like the following:".

File: content/en/docs/event-sources/okta.md (hunk 3 above)
Comment (mstemm) [reply content/en/docs/event-sources/okta.md:16]: Sure, fixed.

File: content/en/docs/event-sources/okta.md (hunk 5 above)
Comment (mstemm) [reply content/en/docs/event-sources/okta.md:34]: The dependencies here aren't quite right. The log entry isn't changed or modified as a part of the configuration, but the rule influences when output appears and what is printed. So I ended up with "For each Okta log entry, the dummy rule would result in falco alerts that look like the following:"

**Review by radhikapc — CHANGES_REQUESTED** [review]
two more.

File: content/en/docs/event-sources/okta.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+# Configuration
+
+See the [README](https://github.com/falcosecurity/plugins/blob/master/plugins/okta/README.md#settings) for information on configuring the plugin. This simply involves providing the `organization/api` token as part of init params. These can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.
+
+The plugin does not use any open params configuration.
+
+# Sample Output
+
+For example, when using a dummy rule like the following:
```
[the remark this answers is not in the document]
Comment (mstemm) [reply content/en/docs/event-sources/okta.md]: Sure, fixed.

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**